# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libsamplerate/libsamplerate-0.1.7-macro-quoting.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- libsamplerate-0.1.7/acinclude.m4.orig	2011-04-26 19:33:01.196077819 +0200
+++ libsamplerate-0.1.7/acinclude.m4	2011-04-26 19:33:48.760077820 +0200
@@ -135,7 +135,6 @@
 	fi
 
 )
-]
 
 if test $ac_cv_c_byte_order = big ; then
 	ac_cv_c_big_endian=1
@@ -154,7 +153,7 @@
 	AC_MSG_WARN([[*****************************************************************]])
 	fi
 
-)# AC_C_FIND_ENDIAN
+])# AC_C_FIND_ENDIAN
 
 
 
